src/code.sh

```
@@ -208,5 +235,9 @@ main() {
         collect_rnaseq_metrics "${sorted_bam_path}" "${REF_FLAT}" "${OUTPUT_DIR}" "${MEM_IN_MB}"
     fi
 
+    if [[ "$run_CollectVariantCallingMetrics" == true ]]; then
+        collect_variant_calling_metrics "${vcf_path}" "${dbsnp_vcf_path}" genome.dict "${OUTPUT_DIR}" "${MEM_IN_MB}"
```

Shellcheck (0.10.0)

[warning] vcf_path is referenced but not assigned.

(SC2154)